python/ray/_private/worker.py

```diff
@@ -673,7 +673,7 @@ def get_objects(self, object_refs: list, timeout: Optional[float] = None):
             object_refs, self.current_task_id, timeout_ms
         )
         debugger_breakpoint = b""
-        for (data, metadata) in data_metadata_pairs:
+        for data, metadata in data_metadata_pairs:
```

Comment:
changes in this file appeared from running `format.sh`, and are not directly related to this PR